calico/src/main/scala/calico/IOWebApp.scala

```diff
@@ -20,15 +20,15 @@ import calico.syntax.*
 import calico.unsafe.given
 import cats.effect.IO
 import cats.effect.Resource
-import org.scalajs.dom
+import fs2.dom.Window
 
 trait IOWebApp:
 
   def rootElementId: String = "app"
 
-  def render: Resource[IO, fs2.dom.HtmlElement[IO]]
+  def render(window: Window[IO]): Resource[IO, fs2.dom.HtmlElement[IO]]
```

Comment:
Instead of passing it as an argument, we could also just have an ambient `def window: Window[IO]`. But I think I like this approach.

Reply:
Hm, maybe I changed my mind 🤔